app/webpacker/packs/school_search.js

```diff
@@ -31,3 +33,60 @@ if (element) {
     },
   });
 }
+
+function getCheckedSchools() {
+  const checkedSchools = sessionStorage.getItem("school-search-checked");
+  if (checkedSchools) {
+    try {
+      return JSON.parse(checkedSchools);
+    } catch (err) {
+      // @todo is there a way of monitoring this?
```

Comment:
google analytics may be the answer here?

Reply:
It's a wider question than just this specific code - right now there isn't any tracking on errors thrown in the front-end, which isn't ideal.

I know google analytics _can_ do it, but it isn't especially good at it. Errors thrown in JS are worded different in every browser, and are also thrown in the language the user has their browser set to, so most error tracking services have learned how to group errors together. I'm pretty sure google analytics doesn't do that.

How is the backend doing it?

(also, this probably isn't a discussion for this PR - maybe one for the dev catchup?)